### PR TITLE
Feat/tooltip: Export Tooltip Component to ui-toolkit

### PR DESCRIPTION
**Pull request by oliveira-gust**

### CODE REVIEW

File: libs/ui-toolkit/tooltip/src/lib/tooltip.component.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  public cssClass = '';
+
+  public ngOnChanges(data) {
+    // Update tooltip css class according to changes in orientation.
+    if (data['orientation']) {
+      if (['above', 'below', 'left', 'right', 'before', 'after'].indexOf(this.orientation) === -1) {
+        this.orientation = 'right';
+      }
+      this.cssClass = this.orientation ? `tooltip-arrow-${this.orientation}` : '';
+    }
```
Comment (DenisOkawa) [inline libs/ui-toolkit/tooltip/src/lib/tooltip.component.ts]: Isn't it better to change the orientation to an Input set and move this logic there?

File: libs/ui-toolkit/tooltip/src/lib/tooltip.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { MatTooltipModule } from '@angular/material/tooltip';
+
+import { TooltipComponent } from './tooltip.component';
+import { By } from '@angular/platform-browser';
+
+fdescribe('TooltipComponent', () => {
```
Comment (guilhermejcgois) [inline libs/ui-toolkit/tooltip/src/lib/tooltip.component.spec.ts]: Can you remove this fdescribe please?

File: libs/ui-toolkit/tooltip/src/lib/tooltip.stories.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import { MatTooltipModule } from '@angular/material/tooltip';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { moduleMetadata, storiesOf } from '@storybook/angular';
+import { text, withKnobs } from '@storybook/addon-knobs';
+
+import { TooltipComponent } from './tooltip.component';
+
+storiesOf('Tooltip', module)
```
Comment (guilhermejcgois) [inline libs/ui-toolkit/tooltip/src/lib/tooltip.stories.ts]: IMHO, great stories for the component! Can you add one more with knobs implemented too?

File: libs/ui-toolkit/tooltip/src/lib/tooltip.component.spec.ts (hunk 2 above)
Comment (oliveira-gust) [reply libs/ui-toolkit/tooltip/src/lib/tooltip.component.spec.ts]: Ooops, sure!

File: libs/ui-toolkit/tooltip/src/lib/tooltip.stories.ts (hunk 3 above)
Comment (oliveira-gust) [reply libs/ui-toolkit/tooltip/src/lib/tooltip.stories.ts]: Thanks! Yeah, I tried implementing one with knobs before pushing, but it didn't work. Remember I mentioned that the tooltip wasn't working when I tried using it with props? :( Not sure if there's a workaround, or if it's worth spending time looking for one.

**Pull request merged**